core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.aarch64.txt

```diff
@@ -32,15 +32,16 @@ container-selinux
 #cri-tools-1.13.0
 curl
 dejavu-sans-fonts # for grafana
-docker-ce-19.03.14
-docker-ce-cli-19.03.14
+docker-ce-20.10.8
+docker-ce-cli-20.10.8
 ebtables
 elasticsearch-oss-7.10.2 # for opendistroforelasticsearch & logging roles
 ethtool
 filebeat-7.9.2
 firewalld
 fontconfig # for grafana
 fping
+fuse-overlayfs # for docker-ce
```

Comment:
It looks like we may need to only add `docker-ce-rootless-extras-20.10.8`. Then `slirp4netns` and `fuse-overlayfs` should be downloaded automatically:

```
[root@tb-dev-rhel-01 ~]# repoquery --requires --resolve --queryformat '%{name}.%{arch}' --archlist=x86_64,noarch docker-ce-rootless-extras-20.10.8
bash.x86_64
slirp4netns.x86_64
docker-ce.x86_64
glibc.x86_64
fuse-overlayfs.x86_64
```

Reply:
Checked for rhel:

```
```
07:33:33 ERROR cli.engine.ansible.AnsibleCommand - fatal: [ec2-3-249-105-114.eu-west-1.compute.amazonaws.com]: FAILED! => {"changed": false, "changes": {"installed": ["containerd.io", "docker-ce-cli-20.10.8", "docker-ce-rootless-extras-20.10.8", "docker-ce-20.10.8"]}, "msg": "Error: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3(FUSE_3.2)(64bit)\nError: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3(FUSE_3.0)(64bit)\nError: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3()(64bit)\n", "rc": 1, "results": ["Loaded plugins: amazon-id, search-disabled-repos, versionlock\nResolving Dependencies\n--> Running transaction check\n---> Package containerd.io.x86_64 0:1.4.9-3.1.el7 will be installed\n--> Processing Dependency: container-selinux >= 2:2.74 for package: containerd.io-1.4.9-3.1.el7.x86_64\n---> Package docker-ce.x86_64 3:20.10.8-3.el7 will be installed\n---> Package docker-ce-cli.x86_64 1:20.10.8-3.el7 will be installed\n--> Processing Dependency: docker-scan-plugin(x86-64) for package: 1:docker-ce-cli-20.10.8-3.el7.x86_64\n---> Package docker-ce-rootless-extras.x86_64 0:20.10.8-3.el7 will be installed\n--> Processing Dependency: slirp4netns >= 0.4 for package: docker-ce-rootless-extras-20.10.8-3.el7.x86_64\n--> Processing Dependency: fuse-overlayfs >= 0.7 for package: docker-ce-rootless-extras-20.10.8-3.el7.x86_64\n--> Running transaction check\n---> Package container-selinux.noarch 2:2.119.2-1.911c772.el7_8 will be installed\n---> Package docker-scan-plugin.x86_64 0:0.8.0-3.el7 will be installed\n---> Package fuse-overlayfs.x86_64 0:0.7.2-6.el7_8 will be installed\n--> Processing Dependency: libfuse3.so.3(FUSE_3.2)(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n--> Processing Dependency: libfuse3.so.3(FUSE_3.0)(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n--> Processing Dependency: libfuse3.so.3()(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n---> Package slirp4netns.x86_64 0:0.4.3-4.el7_8 will be installed\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
```
```